model/hetzner_host.rb

```diff
@@ -31,5 +31,3 @@ def password
 # Indexes:
 #  hetzner_host_pkey                  | PRIMARY KEY btree (id)
 #  hetzner_host_server_identifier_key | UNIQUE btree (server_identifier)
-# Foreign key constraints:
-#  hetzner_host_id_fkey | (id) REFERENCES vm_host(id)
```

Comment:
I don't think the first commit should include these removals. It's just migration for new table

Reply:
This is incorrect.  The migration changes the state of the database, and the model annotations need to match.  Otherwise, the commit would fail CI in isolation.